include/cantera/kinetics/StoichManager.h

```diff
@@ -580,9 +523,9 @@ class StoichManagerN
         }
         bool frac = false;
         for (size_t n = 0; n < stoich.size(); n++) {
+            m_coeffList.push_back(Eigen::Triplet<double>(k[n], rxn, stoich[n]));
```

Comment:
This is a good place for `emplace_back`:
```suggestion
            m_coeffList.emplace_back(k[n], rxn, stoich[n]);
```